java/flight/flight-jdbc-driver/src/main/java/org/apache/arrow/driver/jdbc/ArrowFlightConnection.java

```diff
@@ -96,8 +96,9 @@ private static ArrowFlightSqlClientHandler createNewClientHandler(
           .withPort(config.getPort())
           .withUsername(config.getUser())
           .withPassword(config.getPassword())
-          .withKeyStorePath(config.getKeyStorePath())
```

Comment:
We should continue to support using a keystore/password for two-way SSL.

Reply:
Here I just changed the name that was being used incorrecly since it is supposed to be a trustStore

Reply:
OK. We should create another ticket for 2-way TLS (that would be using a KeyStore and TrustStore and requiring that the server trusts the client).